Src/ILGPU/Backends/OpenCL/CLInstructions.Data.cs

```diff
@@ -157,86 +157,29 @@ partial class CLInstructions
         /// </summary>
         public const string GetLaneIndexOperation = "get_sub_group_local_id()";
 
-        private static readonly string[] MemoryFenceFlags =
-        {
-            "CLK_GLOBAL_MEM_FENCE",
-            "CLK_LOCAL_MEM_FENCE",
-        };
+        private static readonly string[] MemoryFenceFlags = {"CLK_GLOBAL_MEM_FENCE", "CLK_LOCAL_MEM_FENCE",};
```

Comment:
QUESTION: Another example of collapsing well-formatted blocks. Are we able to disable this?